commands/legacy_logger.go

```diff
@@ -0,0 +1,78 @@
+// This file is part of arduino-cli.
+//
+// Copyright 2019 ARDUINO SA (http://www.arduino.cc/)
+//
+// This software is released under the GNU General Public License version 3,
+// which covers the main part of arduino-cli.
+// The terms of this license can be found at:
+// https://www.gnu.org/licenses/gpl-3.0.en.html
+//
+// You can be released from the requirements of the above licenses by purchasing
+// a commercial license. Buying such a license is mandatory if you want to modify or
+// otherwise use the software for commercial activities involving the Arduino
+// software without disclosing the source code of your own applications. To purchase
+// a commercial license, send an email to license@arduino.cc.
+
+package commands
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/arduino/arduino-cli/legacy/builder/constants"
+	"github.com/arduino/arduino-cli/legacy/builder/i18n"
+	"github.com/sirupsen/logrus"
+)
+
+// LegacyLogger wraps logrus but can be used within the `legacy` package
+type LegacyLogger struct{}
+
+// Fprintln is supposed to let the caller decide the target, let's not do it
+// and just forward to Println
+func (l LegacyLogger) Fprintln(w io.Writer, level string, format string, a ...interface{}) {
+	l.Println(level, format, a...)
```

Comment:
I think that this will create some regressions because the parameter `w` must be considered to redirect the output to the correct stream... the legacy `i18n` module is really a mess, let's have a call to plan a way to migrate to a saner logging infra.